src/org/typo3/chefci/v2/stages/Publish.groovy

```diff
@@ -21,6 +21,8 @@ class Publish extends AbstractStage {
      */
     protected publish() {
 
+        (new Slack(script)).notifyVersionBump()
```

Comment:
generally speaking, you could add all "helpers" to the abstract stages - so you have them in all stage implementations...